src/rpc/misc.cpp

```diff
@@ -177,6 +180,8 @@ UniValue getdescriptorinfo(const JSONRPCRequest& request)
 
 UniValue deriveaddresses(const JSONRPCRequest& request)
 {
+    const std::string EXAMPLE_DESCRIPTOR = "\"wpkh([d34db33f/84h/0h/0h]xpub6DJ2dNUysrn5Vt36jH2KLBT2i1auw1tTSSomg8PhqNiUtx8QX2SvC9nrHu81fT41fvDUnhMjEzQgXnQjKEu3oaqMSzhSrHMxyyoEAmUHQbY/0/*)#cjjspncu\"";
```

Comment:
This shouldn't include the escaped ", similar to EXAMPLE_ADDRESS https://doxygen.bitcoincore.org/util_8cpp.html#adbb21c6828f59f3df79b3d25c0821635

Reply:
Thanks for the hint, done.